gulpfile.babel.js

```diff
@@ -308,12 +308,8 @@ gulp.task('mocha:closure', ['closure'], () => {
     .pipe($.rename('temp.html'))
     .pipe(gulp.dest('test'))
     .pipe($.mochaPhantomjs({reporter: 'tap'}))
-    .on('finish', () => {
-      del('test/temp.html', {force: true});
-    })
-    .on('error', () => {
-      del('test/temp.html', {force: true});
-    });
+    .on('finish', () => del.sync('test/temp.html'))
+    .on('error', () => del.sync('test/temp.html'));
```

Comment:
Async del was used here, but it was never indicated that it was done, so switched to using the sync methods. `force: true` was also removed as it has no impact on this.